### PR TITLE
chore: Add my profile to LinkFree

### PR DESCRIPTION
**Pull request by anurag-rajawat**
## Changes proposed
Added `anurag-rajawat.json` file

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2447"></a>

### CODE REVIEW

File: data/anurag-rajawat.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    { "platform": "twitter", "url": "https://twitter.com/anuragrajawat22" },
+    { "platform": "github", "url": "https://github.com/anurag-rajawat" },
+    {
+      "platform": "linkedIn",
+      "url": "https://www.linkedin.com/in/anurag-rajawat22/"
+    }
+  ],
+  "links": [
+    {
+      "name": "GitHub: Let's Collaborate",
```
Comment (srkds) [inline data/anurag-rajawat.json:18]:
```suggestion
      "name": "Let's Collaborate",
```

will look like this 👇
image


Please ignore this suggestion if it was intended.

Comment (anurag-rajawat) [reply data/anurag-rajawat.json]: I missed it. Thanks for pointing out.

**Review by Mahesh-Kasabe — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Niravprajapati1 — APPROVED** [review]
Looks pretty awesome.

**Pull request merged**